dashboard/app/controllers/code_reviews_controller.rb

```diff
@@ -2,7 +2,7 @@ class CodeReviewsController < ApplicationController
   before_action :authenticate_user!
 
   # Make sure we don't forget to authorize each action
-  check_authorization
+  check_authorization except: [:from_peers]
```

Comment:
I think the logic in `from_peers` ensures that only the viewable items are returned but let me know if there's additional cancan authorization I should do for `from_peers`.